AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -1,28 +1,52 @@
 """
-Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 
-Hold constants used across both hydra and non-hydra scripts.
+Holds constants used across both hydra and non-hydra scripts.
 """
 
 # Light type options for the Light component.
 LIGHT_TYPES = {
-    'unknown': 0,
     'sphere': 1,
     'spot_disk': 2,
     'capsule': 3,
     'quad': 4,
     'polygon': 5,
+    'unknown': 0,
```

Comment:
I moved unknown to this position as I discovered the light component inherits type based on shape (which is not removed when deleting the component). This allows me to add the light component, then move it to the "unknown" type. If unknown is in the first position, this value isn't properly tested.

Reply:
dictionary is not an ordered list, you can't be certain that it will behave in this order on another environment.
also why move unknown to the middle and not the end? why not conditional inside the for loop for handling unknown

Reply:
I've set this back in order and just opted to not run 'unknown' in the loop as it's likely P2 anyway.